index.html

```diff
@@ -0,0 +1,193 @@
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="section_2">
+      <div class="tonic_image_1">
+        <img src="images/tonic_image2.png" alt="Tonic image">
+      </div>
+      <div>
+      <div class="tonic_div">
+        <h2>Tonic</h2>
+        <ul>
+          <li class="tonic_title">CANOPY</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">Back End Dev</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">2015</li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="section_2">
+      <div class="tonic_image_1">
+        <img src="images/post_story2.png" alt="Tonic image">
+      </div>
+      <div>
+      <div class="tonic_div">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul>
+          <li class="tonic_title">CANOPY</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">Back End Dev</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">2015</li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about_section">
+      <h2 class="about_title">About Myself
+      </h2>
+        <p>Hello I'm software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="about_social_media1">
+        <h2 class="connect">LET'S CONNECT</h2>
+        <div class="about_social">
+          <a href=""><img src="images/Vector5.png" alt=""></a>
+          <a><img src="images/Linkedin icon.png" alt=""></a>
+          <a><img src="images/Vector4.png" alt=""></a>
+          <a><img src="images/Vector2.png" alt=""></a>
+          <a><img src="images/Vector3.jpg" alt=""></a>
+        </div>
+        <div class="view_button">
+          <button>Get my resume</button>
```

Comment:
- [ ] Kindly make sure this button has the **pressed state** style.
![states](https://user-images.githubusercontent.com/99656687/236605815-4c42a389-0154-4f37-a1ff-fd5354e863ac.png)